spec/unit/analytics/ga4/formatter_spec.rb

```diff
@@ -0,0 +1,116 @@
+require "spec_helper"
+require_relative "../../../../lib/analytics/ga4/formatter"
+require_relative "../../../../lib/analytics/ga4/page_data"
+
+RSpec.describe Analytics::Ga4::Formatter do
+  describe "#consolidated_page_views" do
+    it "doesn't have a slash" do
```

Comment:
```suggestion
    it "ignores input that isn't prefixed with a slash" do
```
Currently the `it` doesn't read correctly as it doesn't describe how the method behaves in that scenario.

Reply:
Same problems can be seen for most of the it statements here, they seem to be written as if they were contexts. Should be pretty straight forward to fix.